shoop/core/fields/tagged_json.py

```diff
@@ -5,6 +5,20 @@
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
 from __future__ import unicode_literals
+
+import datetime
+import decimal
+from enum import Enum
+
+import django.utils.dateparse as dateparse
+from django.core.exceptions import ImproperlyConfigured
+from django.utils import six
+from jsonfield.encoder import JSONEncoder
+from six import text_type
+
+from shoop.utils.importing import load
+from shoop.utils.iterables import first
+
 """
 "Tagged JSON" encoder/decoder.
```

Comment:
This is not correct. The module docstring should go above the imports. Quote from PEP-8:
> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.